src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +28,16 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        if(words == null){
+            return;
+        }
+
+        words.stream()
+                .filter(word -> word.length() >= 12)
+                .distinct()
+                .sorted((word1, word2) -> word2.length() - word1.length())
+                .map(word -> word.toLowerCase())
```

Comment:
```suggestion
                .map(String::toLowerCase)
```
IDE가 경고하는 부분을 확인해보세요